sdk/eventhubs/azure-messaging-eventhubs-stress/src/main/resources/logback.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+  <if condition='isDefined("DEBUG_SHARE")'>
```

Comment:
Is `DEBUG_SHARE` defined somewhere? Didn't find it.

Reply:
Oh, `DEBUG_SHARE` is pre-defined in stress test framework [set env in job.yaml](https://github.com/Azure/azure-sdk-tools/blob/2d7456e53b5bf2efa310c5fb85d6e1a31389ff7a/tools/stress-cluster/cluster/kubernetes/stress-test-addons/templates/_container_env.tpl#L13). If "DEBUG_SHARE" is defined (on AKS), it will output log to fileshares.

Reply:
Consider I am still testing log size for sdk info log, the logback.xml will be changed after I validate/ tune the log. After that, I 'll add more document as a guide.